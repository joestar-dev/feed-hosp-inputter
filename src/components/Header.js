import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

const Header = (props) => (
  <div className="header">
    <div className="container">
      <h1 className="header__title">{props.title}</h1>
      <h2 className="header__subtitle">{props.subtitle}</h2>
      {/* <div>
        <NavLink to="/" activeClassName="is-active" exact={true}>home </NavLink>
        <NavLink to="/feed" activeClassName="is-active" exact={true}>feed </NavLink>
        <NavLink to="/promo" activeClassName="is-active">promo </NavLink>
      </div> */}
    </div>
  </div>
   
);

Header.defaultProps = {
  title: 'Hospital Feed Inputter'
}

export default Header;