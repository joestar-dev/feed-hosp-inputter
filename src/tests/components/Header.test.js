import React from 'react';
import { shallow } from 'enzyme';
// import toJSON from 'enzyme-to-json';
import Header from '../../components/Header';
// import ReactShallowRenderer from 'react-test-renderer/shallow';

test('should render Header component correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
  // expect(wrapper.find('h1').text()).toBe('Hospital Feed Inputter');
  // const renderer = new ReactShallowRenderer();
  // renderer.render(<Header />);
  // expect(renderer.getRenderOutput()).toMatchSnapshot();
  // console.log(renderer.getRenderOutput()) 
});