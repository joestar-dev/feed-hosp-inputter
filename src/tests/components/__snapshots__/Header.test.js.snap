// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component correctly 1`] = `
<div
  className="header"
>
  <div
    className="container"
  >
    <h1
      className="header__title"
    >
      Hospital Feed Inputter
    </h1>
    <h2
      className="header__subtitle"
    />
  </div>
</div>
`;
